types/index.d.ts

```diff
@@ -1632,7 +1632,7 @@ export interface FontSpec {
    * Height of an individual line of text (see MDN).
    * @default 1.2
    */
-  lineHeight: number | string;
```

Comment:
Seems that this still needs to be `number | string`

@param value - The lineHeight to parse (eg. 1.6, '14px', '75%', '1.6em').

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

Reply:
back with it and did some castings where this type is used across the project